src/nidcpower/metadata/functions.py

```diff
@@ -2009,7 +2009,7 @@
                 'name': 'measurements',
                 'size': {
                     'mechanism': 'python-code',
-                    'value': 'self._parse_channel_count()'
+                    'value': 'self._parse_channel_count(session, channel_name)'
```

Comment:
FYI, I'll copy the metadata changes to the original sources after this PR is accepted.

Reply:
Are you planning to regenerate it and copy the output into this change?
(I'm not saying that I'm advocating that, I'm just asking).

Reply:
The internal metadata for NI-Digital is currently stale.
I implemented a change earlier this year to fix it, but had to back it out because it broke niDigitalCXP behavior.

Reply:
All metadata is now updated from internal exports.